docs/asdf/extending/legacy.rst

```diff
@@ -66,6 +66,12 @@ First, the YAML Schema, defining the type as a pair of integers:
     maxItems: 2
     ...
 
+.. note::
+
+   If you are writing code in a local working directory and you wish to use
+   this YAML code, then save it in the local working directory with the 
+   filename of "fraction-1.0.0.yaml".
+
```

Comment:
Can you confirm that this actually makes the example work? Given the mechanism that the hidden code uses I am not sure that it will.

Also, `asdf.CustomType` has been deprecated in favor of `asdf.extension.Converter` hence why this file is `legacy.rst`. A note like this might be more appropriate in `docs/asdf/extending/converters.rst`